### ISSUE TITLE
Motion Consistency Loss for Monocular Visual Odometry with Attention-Based Deep Learning. (arXiv:2401.10857v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#29409]
Motion Consistency Loss for Monocular Visual Odometry with Attention-Based Deep Learning. (arXiv:2401.10857v1 [cs.CV])<br>
https://ift.tt/gJus2TK<br>
<p>Deep learning algorithms have driven expressive progress in many complex
tasks. The loss function is a core component of deep learning techniques,
guiding the learning process of neural networks. This paper contributes by
introducing a consistency loss for visual odometry with deep learning-based
approaches. The motion consistency loss explores repeated motions that appear
in consecutive overlapped video clips. Experimental results show that our
approach increased the performance of a model on the KITTI odometry benchmark.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
